### PR TITLE
cluster-density-v2 test for OVN-IPI-OVN-ETCD-FIPS-ENCR

### PR DESCRIPTION
**Pull request by skordas**
[[GCP-OVN-FIPS-EtcdEncrypt] cluster-density - 120 nodes](https://issues.redhat.com/browse/OCPQE-24544)

### CODE REVIEW

**Comment by skordas** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.16-nightly-x86-cluster-density-v2-etcd-fips-120nodes

**Comment by skordas** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.16-nightly-x86-cluster-density-v2-etcd-fips-120nodes

**Comment by skordas** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.16-nightly-x86-cluster-density-v2-etcd-fips-120nodes

**Comment by skordas** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.17-nightly-x86-cluster-density-v2-etcd-fips-120nodes

**Comment by skordas** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.16-nightly-x86-cluster-density-v2-etcd-fips-120nodes

**Comment by skordas** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.16-nightly-x86-cluster-density-v2-etcd-fips-120nodes

**Comment by skordas** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.16-nightly-x86-cluster-density-v2-etcd-fips-120nodes

**Comment by skordas** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.16-nightly-x86-cluster-density-v2-etcd-fips-120nodes

**Comment by skordas** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.17-nightly-x86-cluster-density-v2-etcd-fips-120nodes

**Comment by skordas** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.16-nightly-x86-cluster-density-v2-etcd-fips-120nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.16-nightly-x86.yaml [hunk 1]
```diff
@@ -54,6 +54,20 @@ tests:
...
     workflow: openshift-qe-installer-gcp-ipi-ovn-etcd-encryption-fips
   timeout: 5h0m0s
+- always_run: false
+  as: cluster-density-v2-etcd-fips-120nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: gcp-qe
+    env:
+      COMPUTE_NODE_REPLICAS: "120"
+      COMPUTE_NODE_TYPE: e2-standard-8
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.16-nightly-x86.yaml:64]: I think 'e2-standard-4' should be enough. I referred to AWS's 120 nodes config, it used COMPUTE_NODE_TYPE: m5.xlarge (4vCPU)
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml#L47

Comment (skordas) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.16-nightly-x86.yaml]: Done

**Pull request merged**